CI/wininstaller/build_installer.cmd

```diff
@@ -0,0 +1,95 @@
+@echo off
+title VCMI Installer Builder
+setlocal enabledelayedexpansion
+cls
+
+REM Define variables dynamically relative to the normalized base directory
+set "AppVersion=1.6.1"
+set "AppBuild=1122334455A"
+set "InstallerArch=x64"

```

Comment:
We also package 32-build for ancient systems

Reply:
If you want you use this CMD script for building I can't turn these variables into CMD script params aswell. 
I was thinking you can use similar method as is here https://github.com/ihhub/fheroes2/blob/2498a89c6f55222032f651812c512da22a45add9/.github/workflows/msvc.yml#L86

Whole installer is written to support both x86 and x64 packaging using iscc.exe params.

Reply:
If this script is just for testing / as example - then fine.

Still needs integration into workflow, similar to fheroes2.

Current installer can be removed & that workflow step should be replaced with simple .zip packaging instead